### ISSUE TITLE
[Feature Request] Accessibility support?

### ISSUE THREAD

**Issue by ethindp** [raysan5/raygui#388]
I'm not sure how easy this would be for an immediate-mode UI, but I thought I'd ask. There is a library called [AccessKit](https://github.com/AccessKit/accesskit) that simplifies this task a lot and abstracts away the platform-specific things you'd need to normally use. The API documentation is mostly in Rust (as the library is written in Rust) but there are C examples. I can help if we can figure out the problem of "it's an immediate-mode UI", but it would be amazing if this could be done. A prerequisite is issue #385 as it will be a requirement for screen readers and individuals who cannot use the mouse.

**Comment by raysan5** [comment]
@ethindp I'm afraid this is really out-of-scope for raygui project. AccessKit seems to be a lower level library, like GLFW (that is integrated into raylib).

**Comment by ethindp** [comment]
@raysan5 May I ask why such a critical feature like this would be out-of-scope? AccessKit might not be the most high-level library but it's much higher level than the OS accessibility abstractions (UIA, AT-SPI2, whatever MacOS implements).

**Comment by raysan5** [comment]
@ethindp Mostly two reasons:

1. raygui is a very small self-contained library, I don't want to add an humungus external dependency.
2. I can't maintain it.

Thanks for your comprehension.

**Comment by raysan5** [comment]
@ethindp Note that raygui is a free and open source project. If you think that feature is crucial for your target project, you can fork it and implement it yourself.

**Comment by ethindp** [comment]
Ah okay, thank you for your explanations.
